test/test_utils.py

```diff
@@ -809,6 +809,10 @@ def test_win_prefix_longpath_is_short(self):
         path = 'C:\\foo\\' + (252 * 'a')
         self.assertEqual(path, win_prefix_longpath(path))
 
+    def test_win_prefix_longpath_unc(self):
+        path = '\\\\server\\' + (251 * 'a')
```

Comment:
```suggestion
        path = r'\\server\' + (251 * 'a')
```

Reply:
This one does not work because of the trailing backslash. `r'\\server\'`  is a syntax error as the trailing backslash is interpreted as escape character for the single quite..

Reply:
Ah yes, forgot about this stupid limitation (raw strings cannot end with a backslash).